### DISCUSSION TITLE
Github desktop and VS Code folders not picking up changes

### DISCUSSION THREAD

**Discussion by GusGF** [community/community#23281] [New to GitHub]
<p>Hi,</p>
<p>I’ve been using Github desktop (GHD) and VS Code as my goto version control. Recently I made a change as to the coverage GHD was keeping. Let me explain, I’m doing a WP course and originally everything under the ‘public’ folder was being monitored, tbh this was overkill. This also caused a problem when trying to update plugins so I changed what GHD was watching to my theme folder where all my work was.  I also disabled the VS Code’s built-in source control (which I’d been using up to now) as I thought to myself I’ve got GHD independently watching the folder concerned. But now when I go into GHD no changes are being picked up as I’m working?!</p>
<p>I created the new respository in GHD on the following directory:</p>
<pre><code>C:\Users\Gus\Local Sites\university\app\public\wp-content\themes\custTheme
</code></pre>
<p>I’ve just noticed in GHD when I chose ‘View the files of your repository in Explorer’ it’s taking me to this directory:</p>
<pre><code>C:\Users\Gus\Local Sites\university\app\public\wp-content\themes\custTheme\University-3
</code></pre>
<p>In the dialogue for creating a new repo I’ve set the following for the path:</p>
<pre><code>C:\Users\Gus\Local Sites\university\app\public\wp-content\themes\custTheme
</code></pre>
<p>Repo name: </p>
<pre><code>University-3
</code></pre>
<p>I would have expected the .git folder to be in ‘custTheme’ not a new nested folder called ‘University-3’. Why is it doing this? Whenever I’ve created a repo manually in the past before using GHD, the .git folder and .gitignore file would be in the directory you actually created the repo in. </p>

**Comment by GusGF** [comment]
<p>Think I may have sorted it. Coming from the console, creating a repo for me is about running the command</p>
<pre><code>git init
</code></pre>
<p> in the directory where you want vc. So the idea of giving your repo a name, as in GHD, is sort of alien for me anyway. I’d only do that sort of thing for a remote.</p>
<p>So I guess in GHD giving a repo a name adds the repo as a directory. I still think it’s counter intuitive and nuts. I mean what if I wanted to create a new repo for an existing project? What I’ve had to do is create the repo in the console and import in GHD.</p>
<p>Love to get input from someone even if it’s a RTFM comment.</p>

**Comment by jcansdale** [comment]
<p>I’ve asked the GitHub Desktop team about this. I’ll let you know what they say.</p>

**Comment by GusGF** [comment]
<p>I’d appreciate that.</p>

**Comment by steveward** [comment, marked as answer]
<p>Hi <a class="mention" href="/gusgf">@gusgf</a>, I work on the GitHub Desktop team so hopefully I can provide some clarity here.</p>
<p>The <code>New repository</code> option in GitHub Desktop is for when you want to create an entirely new repository – it will create a new directory inside of the directory you select in the <code>Local path</code> option (as you have discovered).</p>
<p>The <code>Add local repository</code> option works for when you want to add either an existing project (that has not been initialized as a repository) or an existing repository. When you add a project that has not been initialized as a repository you will see this dialog:</p>
<p><img src="https://user-images.githubusercontent.com/721500/181076025-2b71b689-fc3c-4812-9c7b-c6b83ec3d2c1.png" alt="Screen Shot 2020-04-23 at 4.51.00 PM.png" data-base62-sha1="7QhvGN5eWN66ElqjvEz6aJOfUTL" width="403" height="248"></p>
<p>Clicking the <code>create a repository</code> option will take you through the flow of initializing your existing project as a repository and adding it to GitHub Desktop. You can also trigger this dialog by dragging-and-dropping a project into GitHub Desktop or by using the <code>github</code> command within your project’s directory from the command line.</p>
<p>I hope that helps explain things. Let me know if I can help with any other questions.</p>

**Comment by GusGF** [comment]
<p>Shouldn’t the ‘Create a Repo’ be implemented as a button as this seems logical. I believe you’ve set a context here by having buttons for ‘cancel’ and ‘add repo’ but none for creating.</p>
